sketch.js

```diff
@@ -116,13 +116,14 @@ function sketch_header_background(p) {
       //this.qt.draw();
 
       // the connects
+      p.stroke(255, 255, 255, 1);
       for (const pair of this.connects){
         let d = p.map(distance(pair[0].x, pair[0].y, pair[1].x, pair[1].y), 0, 100, 0.8, 0);
-        p.stroke(255, 255, 255, d);
-        p.strokeWeight(d);
+        //p.stroke(255, 255, 255, d);
+        p.strokeWeight(d * d);
         p.line(pair[0].x, pair[0].y, pair[1].x, pair[1].y);
       }
-      this.connects = [];
+      this.connects.clear();
```

Comment:
can just clear the current set rather than creating a new one, not sure if this makes a difference